assets/sass/modules/_accessibility.scss

```diff
@@ -8,3 +8,13 @@
   height: 0;
   overflow: hidden;
 }
+
+.focused-input,
```

Comment:
Any reason not to use `:focus` on the regular input class instead of applying this via JS?

Reply:
This was already set up to use `.focused-input`.

I originally started with regular input classes with the `:focus` command, but was very un-reliable when it came to the button.

The other part of this, as the screenshots showed, the checkbox remember me has the full width box, and `:focus` on that element does not work.